metadata-service/factories/src/main/java/com/linkedin/gms/factory/graphql/GraphQLEngineFactory.java

```diff
@@ -236,6 +236,8 @@ protected GraphQLEngine graphQLEngine(
     args.setDataProductService(dataProductService);
     args.setGraphQLQueryComplexityLimit(
         configProvider.getGraphQL().getQuery().getComplexityLimit());
+    args.setGraphQLQueryIntrospectionEnabled(
+        configProvider.getGraphQL().getQuery().isIntrospectionEnabled());
```

Comment:
@david-leifker is there a way to test/verify this will work before merging? Just for my knowledge moving forward. I am not sure if existing tests cover this or not?